packages/salesforcedx-vscode-core/test/vscode-integration/telemetry/index.test.ts

```diff
@@ -148,28 +160,41 @@ describe('Telemetry', () => {
       // create vscode extensionContext in which telemetry msg has been previously shown
       mockContext = new MockContext(true);
 
-      await telemetryService.initializeService(mockContext, machineId);
+      await telemetryService.initializeService(
+        mockContext,
+        'ext_name',
+        'testKey007',
+        'v0.0.1'
+      );
 
-      const telemetryEnabled = telemetryService.isTelemetryEnabled();
+      const telemetryEnabled = await telemetryService.isTelemetryEnabled();
       expect(telemetryEnabled).to.be.eql(true);
 
-      telemetryService.showTelemetryMessage();
+      showTelemetryMessage(mockContext);
       assert.notCalled(mShowInformation);
       expect(teleStub.firstCall.args).to.eql([true]);
     });
 
-    it('Should send telemetry data', async () => {
+    xit('Should send telemetry data', async () => {
```

Comment:
I haven't been able to spy or mock the `reporter` from the utils module so all the tests using it are skipped for now. Let me know if you have an idea on what to try to get them working.

Reply:
When util is not webpack'ed unit tests could work with `proxyquire`.

We could tsc the util package files _just_ for unit testing; or we webpack the extensions, and don't webpack util package.

Reply:
Yes, I'll look into that in a follow up PR since I've ran out of ideas on how to make it work when the code is processed by webpack.